tests/contrib/names/test_names_datastreams.py

```diff
@@ -58,11 +42,48 @@ def xml_entry():
         '           </employments:affiliation-group>\n'
         '       </activities:employments>\n'
         '    </activities:activities-summary>\n'
-        '</record:record>\n',
-        encoding="raw_unicode_escape"
+        '</record:record>\n'
     )
 
 
-def test_orcid_xml_transformer(xml_entry, expected_from_xml):
+@pytest.fixture(scope='module')
+def xml_entry():
+    # simplified version of an XML file of the ORCiD dump
+    return XML_ENTRY_DATA
+
+
+@pytest.fixture(scope='module')
+def bytes_xml_entry():
+    # simplified version of an XML file of the ORCiD dump
+    return bytes(XML_ENTRY_DATA, encoding="raw_unicode_escape")
+
+
+@pytest.fixture(scope='module')
+def expected_from_xml():
+    return {
+        'given_name': 'Lars Holm',
+        'family_name': 'Nielsen',
+        'identifiers': [
+            {
+                'scheme': 'orcid',
+                'identifier': 'https://orcid.org/0000-0001-8135-3489'
+            }
+        ],
+        'affiliations': [{'name': 'CERN'}]
+    }
+
+
+def test_orcid_xml_transformer(bytes_xml_entry, expected_from_xml):
     transformer = OrcidXMLTransformer()
-    assert expected_from_xml == transformer.apply(xml_entry)
+    assert expected_from_xml == transformer.apply(bytes_xml_entry)
+
+
+class MockResponse():
+    text = XML_ENTRY_DATA
+    status_code = 200
+
+
+@patch('requests.get', side_effect=lambda url, headers: MockResponse())
```

Comment:
not sure if this is the best way to test it... but is the sanest (i.e. not making external requests to orcid each time we test).